osmaxx/core/templates/pages/about_us.html

```diff
@@ -18,7 +18,7 @@
     <h3>Useful links</h3>
     <p>Can be found in our separate <a href="{% url 'pages:downloads' %}">downloads page</a></p>
 
-    <h3>About Us</h3>
+    <h3>About us</h3>
```

Comment:
The decision in #630 was to use title case for the page titles (`<h2>` in our case) and sentence case for everything else, including section titles. Dunno if the resulting mixture is that good an idea, but it doesn't look too bad.